src/dist/worker.cc

```diff
@@ -112,57 +194,73 @@ agd::Status Worker::Run(const Params& params, const Parameters& aligner_params,
                                  response_queue_address);
   }
 
-  zmq_recv_socket_->setsockopt(ZMQ_SNDHWM, 5);
-  int val = zmq_recv_socket_->getsockopt<int>(ZMQ_SNDHWM);
-  cout << "snd hwm value is " << val << " \n";
+  try {
+    zmq_incomplete_request_socket_->connect(
+        incomplete_request_queue_address.c_str());
+  } catch (...) {
+    return agd::errors::Internal("Could not connect to zmq at ",
+                                 incomplete_request_queue_address);
+  }
+
+  // zmq_recv_socket_->setsockopt(ZMQ_SNDHWM, 5);
```

Comment:
The code is for old sockets only.  I had commented it to see what happens if we do not limit the buffer size. I forgot to uncomment it.

The incomplete request sockets do not need it since the traffic is very less.